Formula/davix.rb

```diff
@@ -22,8 +22,10 @@ class Davix < Formula
   def install
     ENV.libcxx
 
-    system "cmake", ".", *std_cmake_args
-    system "make", "install"
+    cp "release.cmake", "version.cmake"
```

Comment:
Why is the build process changing?

Reply:
There's still a slight bug in the machinery which generates version numbers in 0.7.5: The existence of version.cmake should not have been necessary, but cmake complains if it is not.

0.7.4 could not be built due to this: https://github.com/Homebrew/homebrew-core/pull/41640

I'll make sure to fix this in 0.7.6 (I maintain davix), in the meantime I'm adding a workaround here.

Reply:
Great, thanks for explaining. One tiny other thing: we tend to prefer to build from within the build directory, to maintain our formulas homogeneous:

```
mkdir "build" do
  system "cmake", "..", *std_cmake_args
  system "make", "install"
```

Reply:
Ah, sure. I've updating the formula to use in-source build.